templates/az-oci-adbs/main.tf

```diff
@@ -1,28 +1,46 @@
+
 resource "azurerm_resource_group" "resource_group" {
   location = var.location
   name     = var.resource_group
 }
 
-module "adbs_network" {
-  source = "../../modules/azure-vnet-subnet"
-  providers = {
-    azurerm = azurerm
+
+# Delete comment before merge
+# Reference code : https://github.com/Azure/terraform-azurerm-avm-res-network-virtualnetwork/blob/5b60a9a29fa471d3298427f706645e80f09de9da/examples/legacy_address_prefix/README.md 
+module "avm_odbas_network" {
+  source  = "Azure/avm-res-network-virtualnetwork/azurerm"
+  version = "0.2.4"
```

Comment:
I wonder if the version is a required field here? Shall we remove the hardcoded value and let it use the latest one?

Reply:
yes as of now we can not use latest. We have landing zone's own tfm version restriction hence this has to stay

```
│ Error: Unsupported Terraform Core version
│ 
│   on .terraform/modules/avm-vnet-subnet/modules/subnet/terraform.tf line 2, in terraform:
│    2:   required_version = ">= 1.9.2"
```